CHANGELOG.md

```diff
@@ -3,6 +3,15 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [1.1.3.0](https://github.com/unzerdev/php-sdk/compare/1.1.2.0..1.1.3.0)
+### Added
+*   Enable PHP 8.0 compatibility.
+*   Allow PHPUnit version 8.x and 9.x in composer dev requirementsK and adjust tests accordingly.
```

Comment:
this sentence is rather odd ?
Is this actually two lines merged together?

Reply:
It is mend to be one line since I felt the two point belong together here. Allow the Installation of PHPUnit 8 and 9. And adjust tests to work with those versions. (remove deprecated methods).

Reply:
Ok, just remove the extra `K` then.